src/features/farming/house/House.tsx

```diff
@@ -1,255 +1,286 @@
-import React, { useContext } from "react";
-import { useActor } from "@xstate/react";
-import { Modal } from "react-bootstrap";
-
-import { Context } from "features/game/GameProvider";
-import { GRID_WIDTH_PX } from "features/game/lib/constants";
-import {
-  getLevel,
-  getRequiredXpToLevelUp,
-  upgradeAvailable,
-  SKILL_TREE,
-} from "features/game/types/skills";
-
-import house from "assets/buildings/house.png";
-import smoke from "assets/buildings/smoke.gif";
-import player from "assets/icons/player.png";
-import questionMark from "assets/icons/expression_confused.png";
-import close from "assets/icons/close.png";
-import alert from "assets/icons/expression_alerted.png";
-
-import plant from "assets/icons/plant.png";
-import pickaxe from "assets/tools/stone_pickaxe.png";
-
-import { Action } from "components/ui/Action";
-import { InnerPanel, OuterPanel, Panel } from "components/ui/Panel";
-import { Label } from "components/ui/Label";
-import { ITEM_DETAILS } from "features/game/types/images";
-import { InventoryItemName } from "features/game/types/game";
-import { skillUpgradeToast } from "features/game/toast/lib/skillUpgradeToast";
-import { ToastContext } from "features/game/toast/ToastQueueProvider";
-
-import { SkillUpgrade } from "./components/SkillUpgrade";
-import { SkillTree } from "./components/SkillTree";
-import { homeDoorAudio } from "lib/utils/sfx";
-import { Button } from "components/ui/Button";
-
-export const House: React.FC = () => {
-  const { gameService } = useContext(Context);
-  const { setToast } = useContext(ToastContext);
-  const [gameState] = useActor(gameService);
-  const { state } = gameState.context;
-
-  const [isOpen, setIsOpen] = React.useState(false);
-  const [isSkillTreeOpen, setIsSkillTreeOpen] = React.useState(false);
-  const [isUpgradeAvailable, setIsUpgradeAvailable] = React.useState(false);
-
-  React.useEffect(() => {
-    const upgrades = upgradeAvailable(state);
-    setIsUpgradeAvailable(upgrades);
-    if (upgrades && state.farmAddress) skillUpgradeToast(state, setToast);
-  }, [setToast, state]);
-
-  const openSkillTree = () => {
-    setIsSkillTreeOpen(true);
-  };
-
-  const open = () => {
-    setIsSkillTreeOpen(false);
-    setIsOpen(true);
-    //Checks if homeDoorAudio is playing, if false, plays the sound
-    if (!homeDoorAudio.playing()) {
-      homeDoorAudio.play();
-    }
-  };
-
-  const { gathering, farming } = state.skills;
-
-  const toolLevel = getLevel(gathering);
-  const farmingLevel = getLevel(farming);
-  const totalLevel = toolLevel + farmingLevel;
-
-  const gatheringRequiredXp = getRequiredXpToLevelUp(toolLevel);
-  const farmingRequiredXp = getRequiredXpToLevelUp(farmingLevel);
-
-  const Badges = () => {
-    const BADGES: InventoryItemName[] = Object.keys(SKILL_TREE).map(
```

Comment:
this current code was introduced in #911 . Your incoming change brings back the old code (LN 85). Please rebase from main branch to avoid overwrites. 
```ts
const BADGES: InventoryItemName[] = Object.keys(SKILL_TREE).map(
  (badge) => badge as InventoryItemName
);
```